std/exception.d

```diff
@@ -1807,63 +1797,9 @@ if (isInputRange!IRange)
                 }
             }
         }
-
-        static if (hasSlicing!IRange)
```

Comment:
Should we keep this code inside a `version(none)` block so that we can reenable it once the compiler bug is fixed?